OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/pointsto/AbstractPointsToAnalysis.scala

```diff
@@ -202,6 +200,7 @@ trait AbstractPointsToAnalysis extends PointsToAnalysisBase with ReachableMethod
                     )
                 }
                 handleCall(call, pc)
+                handleIndirectFieldAccesses(pc)
```

Comment:
As this requires no information about the call, why is it invoked for each call individually instead of just once globally for all indirect accesses in the method?

Reply:
How do i know which indirect accesses there are in the method?

Reply:
holy**** (i literally wrote the code myself)